frontend/system/src/Admin/PendingChanges/ListView.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { useEffect } from 'react';
+import { useTranslation } from '@common/intl';
+import {
+  DataTable,
+  NothingHere,
+  TableProvider,
+  createTableStore,
+  useColumns,
+} from '@common/ui';
+import { useBreadcrumbs, useQueryParamsState } from '@common/hooks';
+import { useNavigate } from 'react-router';
+import { RouteBuilder } from '@common/utils';
+import { AppRoute } from 'frontend/config/src';
+
+// TODO: types from codegen
+type PendingChange = {
+  id: string;
+  productName: string;
+  category: string;
+  requestDate: string;
+  changeType: 'Addition' | 'New';
+  requestedBy: string;
+  for: string;
```

Comment:
I guess for this we'll need some mechanism to record this? Perhaps when saving the the entity changes?

Reply:
Yeah... my thinking was that this slots in the middle somewhat? We can save a PendingChange when a drug is added/updated, and then run the upsert entity when this is approved?

Reply:
OH haha I see what you're saying, how are we capturing who the change is for... yeah I guess at the same time of saving the add/edit... perhaps we could have a modal popup on save the entity changes that asks who the change is for?

Reply:
Let's discuss with Jess/Louisa what they want to see when approving. Was assuming it was just the name of the person who entered it.

Reply:
> Yeah... my thinking was that this slots in the middle somewhat? We can save a PendingChange when a drug is added/updated, and then run the upsert entity when this is approved?

I like this approach, will have a think about it. Could have some kind of PendingEntityType in dGraph and return them both or something....

Reply:
> Let's discuss with Jess/Louisa what they want to see when approving. Was assuming it was just the name of the person who entered it.

Oh I think in the mock ups Louisa did, `requestedBy` was the person that entered it and `for` was the country they were making the change for. If it's a little clunky for us to capture that in the UI we could find out actually how useful that info would be for them though?